index.js

```diff
@@ -18,6 +17,10 @@ function log (message) {
 
 class IntegrationAPI extends EventEmitter {
   #driverPath;
+  #integrationInterface;
+  #integrationPort;
+  #integrationHttpsEnabled;
+  #disableMdnsPublish;
```

Comment:
Why do you use class fields?
They only seem to be used inside `init`, then just use local vars.